scripts/release-processor-images.mjs

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env -S node
+
+
+// This script releases indexer processor images to docker hub https://github.com/aptos-labs/aptos-indexer-processors.
+// It does so by copying the images from aptos GCP artifact registry to docker hub.
+
+// Usually it's run in CI, but you can also run it locally in emergency situations, assuming you have the right credentials.
+// Before you run this locally, check one more time whether you can trigger a CI build instead which is usually easier and safer.
+// You can do so via the Github UI or CLI:
+// E.g: gh workflow run copy-processor-images-to-dockerhub.yaml
+//
+// If that doesn't work for you, you can run this script locally:
+//
+// Prerequisites when running locally:
+// 1. Tools:
+//  - docker
+//  - gcloud
+//  - node (node.js)
+//  - crane - https://github.com/google/go-containerregistry/tree/main/cmd/crane#installation
+//  - pnpm - https://pnpm.io/installation
+// 2. docker login - with authorization to push to the `aptoslabs` org
+// 3. gcloud auth configure-docker us-west1-docker.pkg.dev
+// 4. gcloud auth login --update-adc
+//
+// Once you have all prerequisites fulfilled, you can run this script via:
+// GIT_SHA=${{ github.sha }} GCP_DOCKER_ARTIFACT_PROCESSOR_REPO_US="${{ secrets.GCP_DOCKER_ARTIFACT_REPO }}" ./docker/release-processor-images.mjs --language=rust --wait-for-image-seconds=1800
+
```

Comment:
It might be nice to check (when not in CI) that the user has all the necessary tools installed. I see you do it for crane but not the rest like the gcloud CLI.

Reply:
I think mostly this will run in CI